### ISSUE TITLE
Great!

### ISSUE THREAD

**Issue by krisak** [stevenhammerton/vector-desktop#1]
Its great somebody started this effort.

If you write to readme how to properly build it, maybe we could build the aps for platforms and make the screenshots and some info? 

I would like to help.

**Comment by guaraqe** [comment]
I agree! I just did a `npm install` and `./node_modules/.bin/electron .`, but got an error:

```
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: spawn /home/juan/sandbox/vector-desktop/node_modules/electron-prebuilt/dist/electron ENOENT
    at exports._errnoException (util.js:870:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:178:32)
    at onErrorNT (internal/child_process.js:344:16)
    at nextTickCallbackWith2Args (node.js:442:9)
    at process._tickCallback (node.js:356:17)
    at Function.Module.runMain (module.js:443:11)
    at startup (node.js:139:18)
    at node.js:968:3
```

**Comment by krisak** [comment]
@guaraqe Hi, this repo is practicaly dead. Its very old. Try this instead https://github.com/krisak/vector-electron-desktop

**Comment by sn1ckels** [comment]
@krisak That's nice! Could somebody change the link on the official matrix page to the new repo?

**Comment by krisak** [comment]
@sn1ckels well the matrix crowd wanted to wait until our client is more polished. But i had no time to work on it more. I will be more free to do so in September. If you want push foward the project, please read through it and maybe give some ideas about what would you want in client like that. Also there is roadmap if you want to help it would be awesome. I am not experienced with electron.

**Comment by sn1ckels** [comment]
@krisak Currently I'm not that familiar with Electron or Vector.im itself, since I just found out about matrix some days ago, but I'd like to help you as much as I can. I'll look where I can help you :)
